### ISSUE TITLE
Crash with 'NoneType for caret_position' when following hints on some sites

### ISSUE THREAD

**Issue by mbuelte** [qutebrowser/qutebrowser#3395]
I've qutebrowser in version v1.0.3, see below for more details.

Since the last upgrade to one of these packages:
qt5-webkit (5.212.0alpha2-10 -> 5.212.0alpha2-11)
qt5-x11extras (5.9.3-1 -> 5.10.0-1)
qt5-base (5.9.3-1 -> 5.10.0-1)
pyqt5-common (5.9.2-1 -> 5.9.2-2)
python-pyqt5 (5.9.2-1 -> 5.9.2-2)
qt5-xmlpatterns (5.9.3-1 -> 5.10.0-1)
qt5-declarative (5.9.3-1 -> 5.10.0-1)
qt5-location (5.9.3-1 -> 5.10.0-1)
qt5-webchannel (5.9.3-1 -> 5.10.0-1)

I noticed a crash of qutebrowser on some sites, when trying to login there, using follow_hints or mouse press the button.

The both sites I found out having this problem are Jira and Icinga, so I couldn't offer any public url.

Here is the error report:
```
ATTENTION: default value of option force_s3tc_enable overridden by environment.
[1214/083221.867140:WARNING:stack_trace_posix.cc(648)] Failed to open file: /home/xxx/#12321283 (deleted)
  Error: No such file or directory
08:32:32 ERROR: Uncaught exception
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/qutebrowser/browser/webengine/webenginetab.py", line 490, in _js_cb_single
    elem = webengineelem.WebEngineElement(js_elem, tab=self._tab)
  File "/usr/lib/python3.6/site-packages/qutebrowser/browser/webengine/webengineelem.py", line 57, in __init__
    js_dict))
TypeError: Got <class 'NoneType'> for caret_position from JS but expected <class 'int'>: {'attributes': {'data-progress-label': 'Logging in', 'id': 'form_login_btn_submit-zyanqpuxtkic', 'name': 'btn_submit', 'type': 'submit', 'value': 'Login'}, 'caret_position': None, 'class_name': '', 'id': 1, 'outer_xml': '<input type="submit" name="btn_submit" id="form_login_btn_submit-zyanqpuxtkic" value="Login" data-progress-label="Logging in">', 'rects': [{'bottom': 643, 'height': 30, 'left': 827, 'right': 1091, 'top': 613, 'width': 264}], 'tag_name': 'INPUT', 'text': '', 'value': 'Login'}
08:32:46 WARNING: /usr/lib/python3.6/subprocess.py:766: ResourceWarning: subprocess 20958 is still running
  ResourceWarning, source=self)
```

qutebrowser --version:
```
qutebrowser v1.0.3
Git commit: 55815bd17 (2017-11-06 10:53:37 +0100)
Backend: QtWebEngine (Chromium 61.0.3163.140)

CPython: 3.6.3
Qt: 5.10.0
PyQt: 5.9.2

sip: 4.19.6
colorama: no
pypeg2: 2.15
jinja2: 2.10
pygments: 2.2.0
yaml: 3.12
cssutils: 1.0.2 $Id$
attr: 17.3.0
PyQt5.QtWebEngineWidgets: yes
PyQt5.QtWebKitWidgets: yes
pdf.js: 1.9.426 (/usr/share/pdf.js/build/pdf.js)
sqlite: 3.21.0
QtNetwork SSL: OpenSSL 1.1.0g  2 Nov 2017

Style: QFusionStyle
Platform: Linux-4.14.5-1-zen-x86_64-with-arch, 64bit
Linux distribution: Arch Linux (arch)
Frozen: False
Imported from /usr/lib/python3.6/site-packages/qutebrowser
Qt library executable path: /usr/lib/qt/libexec, data path: /usr/share/qt

Paths:
cache: /home/xxx/.cache/qutebrowser
config: /home/xxx/.config/qutebrowser
data: /home/xxx/.local/share/qutebrowser
runtime: /run/user/1000/qutebrowser
system data: /usr/share/qutebrowser
```

**Comment by The-Compiler** [comment]
This was fixed in 180fb0d65aad4509c0fd1e7fbd7168b152638627 and was never part of any v1.0.x release - you're running an outdated git commit:

> Git commit: 55815bd17 (2017-11-06 10:53:37 +0100)

Please update! :wink:

**Comment by mbuelte** [comment]
Wonderful, forgot to check for an qb update. Now it's working again. Thank you!
